sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -23,6 +19,10 @@ use move_vm_runtime::{move_vm::MoveVM, session::Session};
 #[cfg(debug_assertions)]
 use move_vm_types::gas::GasMeter;
 use move_vm_types::loaded_data::runtime_types::Type;
+use std::{
+    collections::{BTreeMap, BTreeSet, HashMap},
+    sync::Arc,
+};
```

Comment:
I have file watchers set, so it will keep changing it even if I revert them 😿 I hope it doesn't create too many of these changes, file watchers generally work best if everyone is using them!

Reply:
IIUC you have something that automatically runs the formatter when the file is changed, (that's the file watcher, right?) If so, I don't think that's the issue here, because we have CI jobs that run the formatter as well.  The issue may be that the formatter your file watcher runs is using different settings than the default formatter settings?